index.html

```diff
@@ -0,0 +1,228 @@
+                <a href="#" class="about">News</a>
+              </div>
+            </div>
+          </div>
+          <span style="font-size:30px;cursor:pointer" onclick="openNav()">&#9776;</span>
+        </div>
+      </div>
+      <div class="heading-div">
+          <div header-div>
+            <div class="heading mobile">
+              <p class="greetings">"Hello! Sharing world"</p>
+              <p class="title">DANMERT NIGERIA <br>ENTERPRISE SUMMIT <br>2019</p>
+            </div>
+            <div class="introduction mobile">
+                <p class="brief-intro">
+                  A joyful celebration believing in the value of openness and sharing, creating a positive change in financial inclusion to the world.
+                </p>
+                <p class="date">2019.10.15(THU) ~ 16(FRI)</p>
+                <p class="national">@ National Stadium Akwa Ibom, Football Center and more</p>
+              </div>
+          </div>
+        </div>
+      </div>
+    </head>
+    <main>
+      <div class="words-presentation" id="main-program">
+        <div class="main-program">
+          <p>Main Program</p>
+        </div>
+
+        <div class="border"></div>
+      
+        <div class="insight-section">
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_01.png" alt="lecture-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Lecture
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_02.png" alt="exhibition-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  DMT Exhibition
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Appreciate various creations applying DMT, organized from football Center Akwa Ibom.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_03.png" alt="forum-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Forum
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Have the time to share your thoughts and opinions with expert for each topic.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_04.png" alt="workshop-icon">
```

Comment:
- [ ] It would be semantic to use the `article` tag instead of the `div` tag for each of the cards in the _**Main Program**_ section. 👍 💻

Reply:
Done